travel_platform/izi_travel_client.py

```diff
@@ -67,11 +74,33 @@ def _prepare_params(self, **kwargs):
         Returns:
             dict: Url parameters.
         """
-        params = {'version': self.api_version, "languages": self.languages}
+        params = {'version': self.API_VERSION, "languages": self.LANGUAGES}
         params.update(kwargs)
         return params
 
+    def _make_request(self, service_url, **params):
+        response = requests.get(service_url,
+                                headers=self.headers, params=params)
+        self._check_response_status(response)
+        return response.json()
+
+
+class IziTravelApiClient(BaseIziTravelApiClient):
+    CITY_ENDPOINT = 'cities'
```

Comment:
This may be done in similar way as it's here:
https://bitbucket.org/atlassian/python-bitbucket/src/c90076b960f8b83b2f9126adcabfef1db5cb89ba/pybitbucket/entrypoints.py?at=master&fileviewer=file-view-default

P.S. just FYI.